main.cpp

```diff
@@ -213,6 +412,34 @@ struct CarWash //                                   1) define an empty struct fo
     3) controll instruments
  */
 
+
+struct Midi       
+{
+    //has value
+    int value = 127; //3) member variables with relevant data types.
+    //has channel
+    int channel = 127; //3) member variables with relevant data types.
+    //has input
+    int input = 127; //3) member variables with relevant data types.
+    //has output
+    int output = 127; //3) member variables with relevant data types.
+    //has sync type
+    int syncType = 127; //3) member variables with relevant data types.
```

Comment:
What kind of information are these properties supposed to represent?
- [ ] revise.  pick properties that provide information and name them accordingly.

Reply:
this is my midi function to play my instrument via midi notes.